include/fmt/format.h

```diff
@@ -1585,68 +1596,79 @@ class specs_setter {
   basic_format_specs<Char> &specs_;
 };
 
+template <typename ErrorHandler> class numeric_specs_checker {
```

Comment:
This seems no longer needed and can be integrated back into `specs_checker` (just revert `specs_checker` changes).

Reply:
Hmm, I think it's still useful because it's used in two places: `dynamic_specs_handler` and `prepared_format` while formatting, to check already prepared specs.

Reply:
Ah, I missed the second use.